Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -334,7 +334,8 @@ namespace gdjs {
       if (!this._action) {
         return true;
       }
-      return !this._action.isRunning();
+      // If this animation is looping, we dont want it to be able to send "true" as it must be endless.
```

Comment:
I still don't understand why `isRunning` would not be enough and we need to add the check on `loop`
Can `isRunning` be false but the animation is not ended?